pom.xml

```diff
@@ -312,4 +336,4 @@
     </plugins>
   </build>
 
-</project>
+</project>
```

Comment:
Don't remove the line break at the end of the file

Reply:
Done